DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: MAESTRO
 Type: Package
 Title: Model-based Analyses of Single-cell Transcriptome and Regulome
-Version: 1.2.1
+Version: 1.2.2
```

Comment:
Please update other fields including 'Date', 'Depends' and etc.

Reply:
If I remember right, the current version should work in R4

Reply:
@taoliu I have updated 'Date' in the `DESCRIPTION` file. But for the R 'Depends', I didn't change it. Because the current `R (>= 3.6.1)` can work in R4. And in the MAESTRO conda environment, `R-base` is fixed to 4.0.2 to avoid incompatibility.